### PR TITLE
[15.0][MIG] account_credit_control: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
TT36416

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CI

**Review by BT-aestebanez — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
CI is still red, @cesar-tecnativa

**Comment by cesar-tecnativa** [comment]
TT36416

**Comment by cesar-tecnativa** [comment]
@pedrobaeza Could you please review?

**Comment by cesar-tecnativa** [comment]
@victoralmau could you please review?

**Comment by cesar-tecnativa** [comment]
@LudLaf could you review this?

File: account_credit_control/views/res_config_settings_view.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                        class="row col-md-6 o_setting_box"
+                        id="credit_control_policy"
+                        title="This allows credit control team to chase overdue payments with different means"
+                    >
+                        <div class="o_setting_left_pane" />
+                        <div class="o_setting_right_pane">
+                            <label
+                                string="Default Policy"
+                                for="credit_control_policy"
+                            />
```
Comment (BT-anieto) [inline account_credit_control/views/res_config_settings_view.xml]: Not existing! This code should not be here.

**Review by BT-anieto — CHANGES_REQUESTED** [review]

**Comment by BT-anieto** [comment]
@cesar-tecnativa credit_control_policy field is not existing. How it is right now it breaks the update.

File: account_credit_control/views/res_config_settings_view.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    </div>
+                    <div
+                        class="row col-md-6 o_setting_box"
+                        id="default_control_tolerance"
+                    >
+                        <div class="o_setting_left_pane" />
+                        <div class="o_setting_right_pane">
+                            <label
+                                string="Default Credit Tolerance"
+                                for="default_control_tolerance"
```
Comment (BT-anieto) [inline account_credit_control/views/res_config_settings_view.xml]: I guess here as well

**Comment by BT-anieto** [comment]
On the other hand, I guess the migrations folder should be deleted.

File: account_credit_control/models/mail_message.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+from odoo import models
+
+
+class MailMessage(models.Model):
+    _inherit = "mail.message"
+
+    def message_format(self):
+        result = super().message_format()
+        credit_control = self.env["ir.model.data"].xmlid_to_res_id(
```
Comment (BT-anieto) [inline account_credit_control/models/mail_message.py]: In the core it has been renamed to **_xmlid_to_res_id**

File: account_credit_control/views/res_config_settings_view.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    </div>
+                    <div
+                        class="row col-md-6 o_setting_box"
+                        id="default_control_tolerance"
+                    >
+                        <div class="o_setting_left_pane" />
+                        <div class="o_setting_right_pane">
+                            <div class="text-muted">
+                              Credit Tolerance Limit
+                          </div>
```
Comment (BT-anieto) [inline account_credit_control/views/res_config_settings_view.xml]: Not clean formated but it was like that in previous version.

File: account_credit_control/views/res_config_settings_view.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                    <div
+                        class="row col-md-6 o_setting_box"
+                        id="credit_control_policy"
+                        title="This allows credit control team to chase overdue payments with different means"
+                    >
+                        <div class="o_setting_left_pane" />
+                        <div class="o_setting_right_pane">
+                            <div class="text-muted">
+                            Default credit control policy to partners
+                        </div>
```
Comment (BT-anieto) [inline account_credit_control/views/res_config_settings_view.xml]: Not clean formated but it was like that in previous version.

**Review by BT-anieto — APPROVED** [review]
LGTM

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

**Review by CarlosRoca13 — APPROVED** [review]

**Review by espo-tony — APPROVED** [review]

**Comment by LudLaf** [comment]
> On the other hand, I guess the migrations folder should be deleted.

Migrations folder now deleted.

**Comment by cesar-tecnativa** [comment]
@pedrobaeza Could you please review?

**Pull request merged**